policyengine_us/variables/gov/states/id/tax/income/deductions/retirement_benefits/id_retirement_benefits.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class id_retirement_benefits(Variable):
```

Comment:
Also rename eligible variable
```suggestion
class id_retirement_benefits_deduction(Variable):
```